crm-project/src/main/java/CRM/service/StatusService.java

```diff
@@ -15,4 +16,9 @@ public class StatusService implements AttributeService {
     public int update(long id, Attribute object) {
         return 0;
     }
+
+    @Override
+    public Status create(Attribute status) {
+        return statusRepository.save((Status) status);
```

Comment:
We are going to check in the future if such status already exists before we push it to the database.
Please add a function that validates if such a name already exists in the DB before you save it.
At the moment, make it throw an exception if the name strings are equal, but in the future we will check percentage of equality. ok?

Reply:
added, let me know if needed for type in the future